distributor-widget/integrations.md

```diff
@@ -78,6 +78,53 @@ Be sure to also enable Enhanced Ecommerce in you Google Analytics under Admin &g
 
 When you have Mews Merchant set up, a payment by a customer is legally required to happen on our domain. Therefore, all the transactions during a checkout are attributed to Mews domain. This is an unfortunate limitation of the checkout process that we cannot currently overcome.
 
+### Conditionally firing tags based on tracking consents
+
+Google Tag Manager integrations are very often used to track users, be it with cookies or other approaches. You can use [TrackingConsents](integrations.md#trackingconsents) and Google Tag Manager to use integrations only when permitted.
+
+Commonly used integration which often needs a tracking consent is Universal Analytics.
+
+Let's see how you can fire Universal Analytics tag only when [TrackingConsents](integrations.md#trackingconsents) permit it.
+
+0. Set up Universal Analytics in Google Tag Manager:
+* See previous section about [Universal Analytics](integrations.md#universal-analytics).
+
+1. Let Distributor know what the tracking consents should be:
+* Call Distributor api methods [enableTracking](reference.md#enableTracking) or [disableTracking](reference.md#disableTracking) when the Distributor widget starts.
+* Calling the api methods at the start guarantees all events have the correct consents.
+* Don't rely on defaults since they can be changed in the future. Defaults are there only for backwards compatibility.
+* You can also call mentioned api calls again later e.g. after user agrees with tracking in your cookie banner.
+
+2. Create data layer variable for performance tracking consent:
+* Open Google Tag Manager.
```

Comment:
🔨 
Is there any way to indent these bullet points? It would be much easier to read it.

![image](https://user-images.githubusercontent.com/46161618/77319925-f3c60780-6d0f-11ea-8e34-5c306c5c7e9b.png)

Reply:
@zdrazil @mjesiolowski Definitely agree with this one, it should be indented.

Reply:
@mjesiolowski @dmarchuk how about now? Please check the markdown preview, it can't be seen in code.

Reply:
@zdrazil Looks good. 👍